live-examples/js-examples/string/string-slice.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">var str = 'The quick red fox jumped over the lazy dog\'s back.';
+console.log(str.slice(30)) // the lazy dog's back.
+console.log(str.slice(4,17)); // quick red fox
+</code>
+</pre>
+    
```

Comment:
Please remove these trailing spaces, and the file should end with a return character.

Reply:
Done